federation/api/install/install.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
```

Comment:
Sorry this is a lot of code and now am wondering if it is worth it to do this or its fine to just add all resources to schema :)

Daniel is doing some refactoring which should reduce this code and ideally, we only want Services so this should be fine.
Probably mention it here (you can add a comment above `package install`) that this code duplication is temporary since we only want to register Services now. When federation will support all v1 resources, we will delete this and just use the kubernetes api registration code.

cc @lavalamp